packages/contracts/CHANGELOG.md

```diff
@@ -36,6 +37,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Refactored 'DAOFactory`to use`PluginSetupProcessor`.
```

Comment:
wrong quote simbol. should be a \` and not a `'`

Reply:
thanks